grails-app/services/org/pih/warehouse/invoice/InvoiceService.groovy

```diff
@@ -233,12 +234,12 @@ class InvoiceService {
             throw new Exception("No order items or order adjustments found for given order")
         }
 
-        order.orderItems.each { OrderItem orderItem ->
+        order.orderItems.findAll { it.orderItemStatusCode != OrderItemStatusCode.CANCELED }.each { OrderItem orderItem ->
```

Comment:
We should add methods to Order to get active orders items and order adjustments since that will likely be a pretty common thing we ask for from an order. If you have time go ahead and make that change now.

Reply:
Done.